Feat: Authenticate to Azure AI with Azure Identity (#2016)
Kong/developer.konghq.com: app/_kong_plugins/ai-proxy-advanced/index.md

    text: OpenAI use cases
  - slug: load-balancing
    text: Load balancing use cases

faqs:
  - q: Can I authenticate to Azure AI with Azure Identity?
    a: |
      Yes, if {{site.base_gateway}} is running on Azure, AI Proxy Advanced can detect the designated Managed Identity or User-Assigned Identity of that Azure Compute resource, and use it accordingly. 

>       In your AI Proxy Advanced configuration, you need to set:

Reviewer: ```suggestion
      In your AI Proxy Advanced configuration, set the following parameters:
```
